website/templates/emails/contributor_added.txt.mako

```diff
@@ -1,7 +1,13 @@
+<%!
+    from website import settings
+%>
+
 Hello ${user.fullname},
 
 ${referrer_name + ' has added you' if referrer_name else 'You have been added'} as a contributor to the project "${node.title}" on the Open Science Framework: ${node.absolute_url}
 
+You will ${'not be ' if all_global_subscriptions_none else ''} automatically subscribed to notification emails for updates to this project. To change your email notification preferences, visit your project or your user settings: ${settings.DOMAIN + "settings/notifications/"}
```

Comment:
@saradbowman Is this to spec? If the user explicitly unsubscribed globally, this sentence is probably unnecessary. It's kinda like a dinner menu that says, "Your hamburger will not be served with cheese."